proto/rollkit/rollkit.proto

```diff
@@ -49,19 +49,19 @@ message Header {
 	// pubkey can't be recovered by the signature (e.g. ed25519).
 	bytes proposer_address = 10;
 
+	//public key of the proposer so we can verify the signature
+	bytes proposer_public_key = 11;
+
 
 	// Hash of block aggregator set, at a time of block creation
-	bytes aggregators_hash = 11;
+	bytes aggregators_hash = 12;
 
 	// Chain ID the block belongs to
-	string chain_id = 12;
+	string chain_id = 13;
 }
 
 message Commit {
-	uint64 height = 1;
-	bytes header_hash = 2;
-	// Note: most of the time this will be a single sinature
-	repeated bytes signatures = 3;
+	repeated bytes signatures = 1;
```

Comment:
you can move `repeated bytes signatures` to `SignedHeader` as it is the only field in the `Commit`